lib/os_cap.c

```diff
@@ -675,7 +680,7 @@ os_cap_get_mba_ctrl(const struct pqos_cap *cap,
                 if (ret == PQOS_RETVAL_OK) {
                         fd = resctrl_alloc_fopen(grp, "schemata", "w");
                         if (fd != NULL) {
-                                fprintf(fd, "MB:0=2000\n");
+                                fprintf(fd, "MB:0=%d\n", vconfig->mba_max * 2);
```

Comment:
Compilation warning
`os_cap.c:683: warning: %d in format string (no. 1) requires 'int' but the argument type is 'unsigned int'.`
```suggestion
                                fprintf(fd, "MB:0=%u\n", vconfig->mba_max * 2);
```